bpf/bpf_overlay.c

```diff
@@ -149,6 +149,36 @@ static __always_inline int handle_ipv6(struct __ctx_buff *ctx,
 not_esp:
 #endif
 
+#if defined(ENABLE_EGRESS_GATEWAY_COMMON)
+	{
+		__u32 egress_ifindex = 0;
+		union v6addr snat_addr, daddr;
+
+		daddr = ip6->daddr;
```

Comment:
This grabs the pod-to-world traffic as it exits the overlay network on the gateway node. Does *another* EGW policy lookup to obtain the desired `egressIP`, does a FIB lookup (to get the corresponding network interface), and redirects straight to that interface.